ocaml/typing/layouts.mli

```diff
@@ -26,6 +26,12 @@ module Sort : sig
       (** Standard ocaml value representation *)
     | Float64
       (** Unboxed 64-bit floats *)
+    | Word
+      (** Unboxed native-size integers (no tag bit) *)
+    | Bits32
+      (** Unboxed 32-bit integers *)
+    | Bits64
+      (** Unboxed 64-bit integers (no tag bit) *)
```

Comment:
I think "no tag bit" is a synonym for "unboxed", and maybe could be dropped (or if it seems to add something to you, probably so too for `Bits32`?).